conda_smithy/lint_recipe.py

```diff
@@ -11,10 +11,16 @@
 import jinja2
 import ruamel.yaml
 
-from conda_build.metadata import ensure_valid_license_family, FIELDS
+from conda_build.metadata import (ensure_valid_license_family,
+                                  FIELDS as cbfields)
 
+FIELDS = cbfields.copy()
```

Comment:
This is a shallow copy, so changing the lists will still change the lists inside conda_build, no?